rubocop/rubocop_rails.yml

```diff
@@ -3,11 +3,34 @@ inherit_from: rubocop.yml
 AllCops:
   Exclude:
     - db/schema.rb
+    - 'bin/**/*'
+    - 'config/**/*'
     - 'vendor/**/*'
 
 Rails:
   Enabled: true
 
+Metrics/BlockLength:
+  Exclude:
+    - 'db/seeds.rb'
+    - 'spec/**/*'
+
+Metrics/LineLength:
+  Exclude:
+    - 'db/seeds.rb'
+
 Metrics/MethodLength:
   Exclude:
     - 'db/migrate/**/*'
+
+Metrics/ModuleLength:
+  Exclude:
+    - 'db/seeds.rb'
```

Comment:
In spirit, I'd prefer keeping this in, and warning that a long seeds file is probably a smell that should be addressed. Ideally, by not having so much stuff needed to be seeded in, but alternatively, by breaking up into separate POROs that handle the various domain elements that must be decomposable if you have seeds running over 100 lines.